zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -197,8 +199,9 @@ object Server {
   val paranoidLeakDetection: UServer                 = LeakDetection(LeakDetectionLevel.PARANOID)
   val disableKeepAlive: UServer                      = Server.KeepAlive(false)
   val consolidateFlush: UServer                      = ConsolidateFlush(true)
-  def unsafePipeline(pipeline: ChannelPipeline => Unit): UServer          = UnsafeChannelPipeline(pipeline)
-  def enableObjectAggregator(maxRequestSize: Int = Int.MaxValue): UServer = ObjectAggregator(maxRequestSize)
+  def unsafePipeline(pipeline: ChannelPipeline => Unit): UServer              = UnsafeChannelPipeline(pipeline)
+  def enableObjectAggregator(maxRequestSize: Int = Int.MaxValue): UServer     = ObjectAggregator(maxRequestSize)
+  def withServerOption(channelOptionList: List[ServerChannelOption]): UServer = ServerOption(channelOptionList)
```

Comment:
I think like `unsafePipeline` if we have `unsafeBootstrap` it would be better. 
That way one can configure the serverBootstrap anyway they like.